### PR TITLE
[#313] Gradle 7.3

### PR DESCRIPTION
**Pull request by ccjernigan**
In addition to updating the version of Gradle, I've added the sha256 to enable verification of the Gradle download

### CODE REVIEW

**Comment by ccjernigan** [comment]
We should wait until PR #320 is merged and we can rebase, to verify that this change doesn't break the CI build.

**Review by pacu — APPROVED** [review]

**Pull request merged**
